aws_lambda_powertools/utilities/parser/models/cloudformation_custom_resource.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict, Type, Union
+
+from pydantic import BaseModel, Field, HttpUrl
+
+from aws_lambda_powertools.utilities.parser.types import Literal
+
+
+class CloudFormationCustomResourceBaseModel(BaseModel):
+    request_type: str = Field(..., alias="RequestType")
+    service_token: str = Field(..., alias="ServiceToken")
+    response_url: HttpUrl = Field(..., alias="ResponseURL")
+    stack_id: str = Field(..., alias="StackId")
+    request_id: str = Field(..., alias="RequestId")
+    logical_resource_id: str = Field(..., alias="LogicalResourceId")
+    resource_type: str = Field(..., alias="ResourceType")
+    resource_properties: Union[Dict[str, Any], Type[BaseModel]] = Field(..., alias="ResourceProperties")
```

Comment:
Hi @ran-isenberg! According to the documentation, this field is not required, so it should be Optional, right?
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/crpg-ref-requests.html

>ResourceProperties
This field contains the contents of the Properties object sent by the template developer. Its contents are defined by the custom resource provider.
Required: No
Type: JSON object

Reply:
yeah i'l fix it.